inc/usb_ccid.h

```diff
@@ -0,0 +1,71 @@
+#include <stdint.h>
+
+#ifndef _USB_CCID_H_
+#define _USB_CCID_H_
+#ifdef __cplusplus
+    extern "C" {
+#endif
+
+#define USB_CLASS_CCID                   0x0b    /**<\brief Smart Card Device class */
+#define USB_DTYPE_FUNCTIONAL_DESC   0x21   /**<\brief Funcitonal descriptor.*/
```

Comment:
Is there any reasons to keep this definition?

Reply:
I misunderstood your first request. I`ll remove it.